assets/js/hooks/disable_zoom_on_input_focus.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * A hook that disables the auto-zoom behavior when focusing an
+ * input on a touch device.
+ *
+ * It is important that this should not prevent users from manually
+ * zooming if they wish. There isn't a portable solution to this
+ * problem, so this hook is a no-op if the detected device is not
+ * known to behave well.
+ *
+ * See: https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone
+ */
+const DisableZoomOnInputFocus = {
```

Comment:
I don't think we need a hook in this case, since we just want to run this once globally. I would move this to a function, perhaps in `events.js` and invoke after `registerGlobalEventHandlers` in `app.js`.